src/core/ticker/Ticker.js

```diff
@@ -107,34 +107,31 @@ class Ticker
          */
         this.started = false;
 
-        // Bind tick
-        this._tick = this._tick.bind(this);
-    }
+        /**
+         * Internal tick method bound to ticker instance.
+         * This is because in early 2015, Function.bind
+         * is still 60% slower in high performance scenarios.
+         * Also separating frame requests from update method
+         * so listeners may be called at any time and with
+         * any animation API, just invoke ticker.update(time).
+         *
+         * @private
+         */
+        this._tick = (time) => {
```

Comment:
Yes, arrow functions (`() => {}`)retain the scope they were declared in, and prevent the `this` headache that people used to get and forced us to use `Function.bind` all over the place.